fighthealthinsurance/utils.py

```
@@ -69,6 +69,68 @@ def is_convertible_to_int(s):
+    Merges consecutive messages with the same role into a single message.
+
+    This is important for LLM APIs that require strict user/assistant alternation.
+
+    Args:
+        history: List of chat messages with 'role' and 'content' keys
+
+    Returns:
+        List of messages with proper alternation (user/assistant/user/assistant...)
+    """
+    if not history:
+        raise Exception("History can not be empty")
```

Ruff (0.14.10)

Create your own exception

(TRY002)